BuildLoader.py

```diff
@@ -978,6 +981,12 @@ def pre_build(self):
 			setattr(self._board, '%s_SIZE' % each, get_fsp_size(fsp_bin) if self._board.HAVE_FSP_BIN else 0)
 			setattr(self._board, '%s_UPD_SIZE' % each, get_fsp_upd_size(fsp_bin) if self._board.HAVE_FSP_BIN else 0)
 
+		if self._board.BUILD_CSME_UPDATE_DRIVER:
```

Comment:
Can we enable BUILD_CSME_UPDATE_DRIVER on QEMU by default even if actual validation is not available?

Reply:
build driver need a csme update library from CSME kit

Reply:
> Please run PatchCheck.py - 'python BaseTools/Scripts/PatchCheck.py'

ran the check, for python files, it is complaining as tab character used, not sure what to do for that

Reply:
Okay with Tabs in python. Let’s clean up that later.